airbyte-cdk/python/airbyte_cdk/destinations/destination.py

```diff
@@ -28,8 +28,11 @@ def write(
     ) -> Iterable[AirbyteMessage]:
         """Implement to define how the connector writes data to the destination"""
 
-    def _run_check(self, config: Mapping[str, Any]) -> AirbyteMessage:
-        check_result = self.check(logger, config)
+    def _run_check(self, config: Mapping[str, Any], configured_catalog: Optional[ConfiguredAirbyteCatalog] = None) -> AirbyteMessage:
+        if configured_catalog:
+            self.check_with_catalog(logger, config, configured_catalog)
```

Comment:
should this be `check_result =` ?

Reply:
ahh yup good catch, I think I caught this in the [pr to follow this](https://github.com/airbytehq/airbyte/pull/24250/files#diff-342c2e5fdce4cd0711458e727948124155a6b175bba4dbdcdcce5094bbd9781a), but I'll add it here